src/main/java/org/broadinstitute/hellbender/tools/funcotator/metadata/VcfFuncotationMetadata.java

```diff
@@ -14,7 +15,8 @@
  */
 public class VcfFuncotationMetadata implements FuncotationMetadata {
 
-    private LinkedHashMap<String, VCFInfoHeaderLine> fieldNameToHeaderLineMap;
+    @VisibleForTesting
+    public LinkedHashMap<String, VCFInfoHeaderLine> fieldNameToHeaderLineMap;
```

Comment:
Can this be package-private instead of public?

Reply:
I was testing with this visible to try to determine why a test was failing.  It was due to a lack of equals/hashcode methods in `VcfFuncotationFactory`.  This was erroneously left in.

I'll revert the visibility of this field to `private`.